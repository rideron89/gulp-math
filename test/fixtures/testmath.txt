gulpcalc(a + 5);
asdf
asdf
gulpcalc(b + 5);