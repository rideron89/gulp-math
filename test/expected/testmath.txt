10
asdf
asdf
15